### PR TITLE
done

### PR DESCRIPTION
**Pull request by Sesue2002**

### CODE REVIEW

File: src/test/java/Problem2Test.java [hunk 1]
```diff
@@ -8,7 +8,27 @@
...
+        int answer1[] = {3};
+
+        Problem2.bubbleSort(input1);
+        assertArrayEquals(answer1, input1);
+
+        //test 3
+        int input0[] = {0};
+
+        Problem2.bubbleSort(input0);
+
```
Comment (pdgetrf) [inline src/test/java/Problem2Test.java:32]: great tests.

let's try to add more tests especially for edge cases such as empty array, null array, etc.

also let's use a loop to go through all the tests without having to repeatedly write the code to call sort and assert.

**Review by pdgetrf — APPROVED** [review]
LGTM
